src/uct/ib/ud/verbs/ud_verbs.c

```diff
@@ -402,7 +402,8 @@ uct_ud_verbs_iface_poll_rx(uct_ud_verbs_iface_t *iface, int is_async)
 
     UCT_IB_IFACE_VERBS_FOREACH_RXWQE(&iface->super.super, i, packet, wc, num_wcs) {
         if (!uct_ud_iface_check_grh(&iface->super, packet,
-                                    wc[i].wc_flags & IBV_WC_GRH, wc[i].sl)) {
+                                    wc[i].wc_flags & IBV_WC_GRH,
+                                    wc[i].sl & 0x7)) {
```

Comment:
minor: `& ntohl(UCT_IB_MLX5_CQE_SL_PKTYPE_MASK)`

Reply:
afaiu, we do not use MLX5 constants in verbs

Reply:
is this something would make sense to move to common ib_device.h?

Reply:
ok, will move there